ast/src/asm_analysis/utils.rs

```diff
@@ -1,3 +1 @@
 
-
```

Comment:
@chriseth Can this file be removed?

Reply:
haha yes, was probably two independent refactors.